serde/src/test/kotlin/com/amazon/ionhiveserde/objectinspectors/IonStructToMapObjectInspectorTest.kt

```diff
@@ -107,4 +122,13 @@ class IonStructToMapObjectInspectorTest {
         return struct
     }
 
-}
\ No newline at end of file
+    private fun makeCaseInsensitiveStruct(): IonStruct {
+        val rawStruct = ION.newEmptyStruct()
+        val struct = IonStructCaseInsensitiveDecorator(rawStruct)
+
+        struct.add("a", ION.newInt(1))
+        struct.add("b", ION.newNull())
+
+        return struct
+    }
+}
```

Comment:
I'd rather see CaseInsensitiveDecoratorTest's `struct_for` extracted to a commonly-accessible place. That pattern is easier to read at a glance.

Same applies to `makeStruct()`, if you could :)

Reply:
Extracted to Util.kt - [220c1e7](https://github.com/amzn/ion-hive-serde/pull/91/commits/220c1e77356f0d30488e0cd5a86201e997b7f336)